code/modules/jobs/job_types/_job.dm

```diff
@@ -253,6 +253,7 @@
 	back = /obj/item/storage/backpack
 	shoes = /obj/item/clothing/shoes/sneakers/black
 	box = /obj/item/storage/box/survival
+	var/skillchip = /obj/item/organ/cyberimp/skillChip
```

Comment:
Will this not give everyone a blank / broken skillchip rather than nothing?

Reply:
Also why even have this var if the thing used is cyberimps anyway?

Reply:
I'm planning to, after this, move implants to cyberimps

Reply:
That's more vestige code.

Reply:
That's vestige code. Outfit is where I thought you were referring to